matrix/test_unittest_matrix.py

```diff
@@ -11,12 +11,12 @@
 
 
 class TestMatrixCreate(unittest.TestCase):
-    """Test the creation of a matrix and that an instance can't be modified."""
-
     def test_initialization(self):
         """Test init function of Matrix."""
-        self.assertIsInstance(Matrix(((1, 2), (3, 4))), Matrix)
-        self.assertIsInstance(Matrix(((1, 2), (5.5, 4))), Matrix)
+        matrix_data = (19, 48), (19, 67)
+        m = Matrix(matrix_data)
+
+        self.assertEqual(str(m), str(matrix_data))
```

Comment:
you got a different test to check str later.
If you only want to chechk the object was created correctly, why not compare m.data with matrix_data?